packages/common/amplify_db_common/pubspec.yaml

```diff
@@ -1,29 +1,35 @@
 name: amplify_db_common
-description: Common utilities for working with databases such as sqlite.
+description: Common utilities for working with databases such as SQLite.
 version: 0.1.0
 homepage: https://github.com/aws-amplify/amplify-flutter/tree/next
 repository: https://github.com/aws-amplify/amplify-flutter/tree/next/packages/common/amplify_db_common
 issue_tracker: https://github.com/aws-amplify/amplify-flutter/issues
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
+  flutter: ">=3.0.0"
 
 dependencies:
-  amplify_core: ">=1.0.0-next.0+1 <1.0.0-next.1"
-  aws_common: ">=0.2.3 <0.3.0"
-  async: ^2.8.0
+  amplify_db_common_dart: ^0.1.0
   drift: ^2.2.0
-  meta: ^1.7.0
-  path: ^1.8.0
-  sqlite3: ^1.9.0
+  flutter:
+    sdk: flutter
+  path: ^1.8.2
+  path_provider: ^2.0.11
 
 dev_dependencies:
   amplify_lints:
     path: ../../amplify_lints
-  build_test: ^2.0.0
-  build_runner: ^2.2.1
-  build_web_compilers: ^3.2.3
-  drift_dev: ^2.2.0+1
-  example_common:
-    path: ../../example_common
-  test: ^1.16.0
+  flutter_test:
+    sdk: flutter
+
+flutter:
+  plugin:
+    platforms:
+      android:
+        package: com.amazonaws.amplify.amplify_db_common
+        pluginClass: AmplifyDbCommonPlugin
+      linux:
+        pluginClass: AmplifyDbCommonPlugin
+      windows:
+        pluginClass: AmplifyDbCommonPlugin
```

Comment:
what about MacOS?

Reply:
Not needed for iOS/macOS - we just want the podspec. Probably not the others either, but I'm not sure how to remove them yet.